songs/models.py

```diff
@@ -0,0 +1,58 @@
+from django.db import models
+from django.contrib.auth import get_user_model
+
+# Create your models here.
+
+class Artist(models.Model):
+    name = models.CharField(max_length=20)
+    
+    def __str__(self):
+        return self.name
+
+
+class Album(models.Model):
+    name = models.CharField(max_length=100)
+    album_artist = models.ForeignKey(Artist, related_name='album_artist', on_delete=models.CASCADE)
```

Comment:
replace ```album_artist``` with ```artist``` and related name should be "albums" instead of "album_artist". Also, read up on related name just to understand the exact purpose of it